vars/signStageMavenRelease.groovy

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
Rename library to `publishToMaven.groovy` being consistent with all other publishing libraries. Also no need to specify the signing part, it is mandatory so just publishToMaven should be fine.